recommendation_engine/config/__init__.py

```diff
@@ -1 +0,0 @@
-"""Contains the configuration constants and classes for the service."""
```

Comment:
Un-delete this.

Reply:
This is in the other PR https://github.com/fabric8-analytics/fabric8-analytics-npm-insights/pull/77/files